sys/userlib/src/macros.rs

```diff
@@ -42,7 +42,17 @@ cfg_if::cfg_if! {
                 { let _ = riscv_semihosting::hprintln!($s, $($tt)*); }
             };
         }
-    } else if #[cfg(feature = "log-null")] {
+    } else if #[cfg(feature = "log-stringbuf")] {
+        #[macro_export]
+        macro_rules! sys_log {
+            ($s:expr) => {
+                { let _ = ringbuf::stringbuf_entry_root!(format_args!(concat!("\0", concat!($s, "\n\r")))); }
```

Comment:
Why does this branch not add any timestamps?

Reply:
Good catch.